linux_os/guide/system/accounts/accounts-banners/banner_etc_issue/rule.yml

```diff
@@ -124,7 +124,6 @@ references:
     stigid@sle12: SLES-12-010030
     stigid@sle15: SLES-15-010020
 
-platform: machine
```

Comment:
I don't see the accounts rules to have `system_with_kernel` platform, you just remove `machine`. Maybe you forgot to add `system_with_kernel` into `linux_os/guide/system/accounts/group.yml`?

Reply:
I have fixed this now